public/apps/configuration/panels/tenant-list/tenant-list.tsx

```diff
@@ -129,10 +128,6 @@ export function TenantList(props: TenantListProps) {
     ));
   };
 
-  if (!props.config.multitenancy.enabled) {
```

Comment:
What do you think about fixing the message [here](https://github.com/opensearch-project/security-dashboards-plugin/blob/main/public/apps/configuration/panels/tenant-list/tenant-list.tsx#L62-L63) for the case where multitenancy is not explicitly disabled in yaml, but the dynamic setting is disabled?

Reply:
Sure - made a quick fix here: [78fb90f](https://github.com/opensearch-project/security-dashboards-plugin/pull/1960/commits/78fb90f813a8a5841cad34f9000877a88f3b55cf)

Reply:
Changed the messaging here: [fd09f20](https://github.com/opensearch-project/security-dashboards-plugin/pull/1960/commits/fd09f2084d54c33e9003b52e0c92c32ffa72c08e), it is not strictly true that it must be enabled to create/edit so I just directed them to turn it on instead.